src/components/MovieView.js

```diff
@@ -33,16 +37,16 @@ class MovieView extends Component {
                         <img className="single-view-poster" src={this.props.selectedMovie.poster_path} alt="movie poster"/>
                         <div className='movie-info'>
                             <h2 className='single-view-title'>{this.props.selectedMovie.title}</h2>
-                            <p>{this.props.selectedMovie.average_rating}</p>
-                            <p>{this.props.selectedMovie.release_date}</p>
-                            <p>{this.props.selectedMovie.genre}</p>
-                            <p>{this.props.selectedMovie.budget}</p>
-                            <p>{this.props.selectedMovie.revenue}</p>
                             <p>{this.props.selectedMovie.runtime}</p>
+                            <p>Rating: {this.props.selectedMovie.average_rating} / 10</p>
+                            {this.props.selectedMovie.release_date && <p>Release Date: {this.props.selectedMovie.release_date}</p>}
+                            {this.props.selectedMovie.genre && <p>Genre: {this.props.selectedMovie.genre}</p>}
+                            {this.props.selectedMovie.budget && <p>Budget: {this.props.selectedMovie.budget}</p>}
+                            {this.props.selectedMovie.revenue && <p>Revenue: {this.props.selectedMovie.revenue}</p>}
```

Comment:
Let's find out whether we can do this in the project without being docked points! Good use of conditional rendering though IMO.